pkgs/misc/uboot/default.nix

```diff
@@ -607,6 +607,14 @@ in {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
+  ubootTuringRK1RK3588 = buildUBoot {
```

Comment:
Should this be `ubootTuringPiRK1` instead?

Reply:
I'm curious to hear about your rationale here.

Though, the names *should* (but they don't always) reflect the defconfig name.

While it's a mouthful to type, I think this name is appropriate.

Reply:
Is this for me or flokli?

Reply:
I oriented myself on how `roc-pc-rk3399_defconfig` comes up with its name, but it's indeed a bit verbose. I changed it to `ubootTuringRK1` (without the `Pi` prefix) - it seems that's what we did for all `ubootRock*` too.

It's called "Turing [Machines] RK1" in various places throughout the kernel. The "Pi" name seems to be used for the carrier board (as it can also hold CM4s).